packages/application-shell/src/components/application-shell-authenticated/new-main-navigation-flag-wrapper.tsx

```diff
@@ -2,7 +2,9 @@
 import { useFeatureToggle } from '@flopflip/react-broadcast';
 import { MAIN_NAVIGATION } from '../../feature-toggles';
 
-const NewMainNavigationFlagWrapper = (props: { children: Function }) => {
+const NewMainNavigationFlagWrapper = (props: {
```

Comment:
Maybe we can add a comment here that this component is intended to be temporary and we will remove it once the new navigation has been rolled out.

Reply:
It seems this comment has already been added by @chloe0592 https://github.com/commercetools/merchant-center-application-kit/blob/2cb7e8227d7406a2c78addd78181f0f3dac66e30/packages/application-shell/src/components/application-shell-authenticated/new-main-navigation-flag-wrapper.tsx#L1

Reply:
Oops! Didn't see the comment in the PR.

Thanks!